boring-sys/patches/boring-pq.patch

```diff
@@ -4558,10 +4994,89 @@ index 09a9ad380..f7d2226e3 100644
    uint8_t x25519_private_key_[32];
 -  KYBER_private_key kyber_private_key_;
 +  KYBER512_private_key kyber_private_key_;
++};
++
++class IPDWingKeyShare : public SSLKeyShare {
++ public:
++  IPDWingKeyShare() {}
++
++  uint16_t GroupID() const override { return SSL_CURVE_IPDWING; }
++
++  bool Generate(CBB *out) override {
++    uint8_t entropy[IPDWING_GENERATE_KEY_BYTES];
```

Comment:
"seed" is better aligned with with the draft.
```suggestion
+    uint8_t entropy[IPDWING_GENERATE_SEED_BYTES];
```

Reply:
But not with the existing patch.